core/src/database/configuration/organization_configuration_key/organization_configuration_key.enum.ts

```diff
@@ -25,6 +25,10 @@ export enum OrganizationConfigurationKeyCode {
       * Alterate guilds of the organization that names and roles will be synced to.
       */
     ALTERNATE_DISCORD_GUILD_SNOWFLAKES = "ALTERNATE_DISCORD_GUILD_SNOWFLAKES",
+    /**
+     * Disocrd Guuild Text Channel Id to post report cards to.
```

Comment:
Typo, also, is this comment accurate?

```suggestion
     * Discord Guild Text Channel Id to post report cards to.
```